nestkernel/CMakeLists.txt

```diff
@@ -152,6 +152,7 @@ target_include_directories( nestkernel PRIVATE
     ${PROJECT_SOURCE_DIR}/thirdparty
     ${PROJECT_SOURCE_DIR}/libnestutil
     ${PROJECT_BINARY_DIR}/libnestutil
+    ${PROJECT_SOURCE_DIR}/models
```

Comment:
How is this related to the changes of this PR?

Reply:
This is required because of the [added](https://github.com/nest/nest-simulator/blob/f91ab2b41cc864bbc151e413bb26c3d0dcaca868/nestkernel/common_synapse_properties.cpp#L31-L32) `#include "weight_recorder.h"` in `common_synapse_properties.cpp`.